src/server/command_registry.cc

```diff
@@ -54,12 +55,19 @@ bool CommandId::IsTransactional() const {
 }
 
 void CommandId::Invoke(CmdArgList args, ConnectionContext* cntx) const {
-  uint64_t before = absl::GetCurrentTimeNanos();
-  handler_(std::move(args), cntx);
-  uint64_t after = absl::GetCurrentTimeNanos();
-  auto& ent = command_stats_[ServerState::tlocal()->thread_index()];
+  ServerState* ss = ServerState::tlocal();
+  int64_t before = absl::GetCurrentTimeNanos();
+  handler_(args, cntx);
+  int64_t after = absl::GetCurrentTimeNanos();
+  int64_t execution_time_micro_s = (after - before) / 1000;
+  auto& ent = command_stats_[ss->thread_index()];
+  if (execution_time_micro_s > ss->log_slower_than_usec) {
```

Comment:
if log_slower_than_usec is negative you dont what to add to slow log as it is disabled